index.md

```diff
@@ -195,96 +129,113 @@ either the result of a digital migration or information that enables the creatio
-![Information Package structure](figs/visio/fig_1_ip_structure.svg "Structure of an IP compliant with the E-ARK Common Specification for IPs structure")
+*Definition:* *Segmentation* is a physical manifestation of a logical AIP where a set of 
+physical container files contains parts of the logical AIP. Each segment of the logical AIP 
+is a packaged as a TAR or ZIP file and contains its own structural metadata.
 
-**Figure 1:**
-Structure of an IP.
+When the AIP is segmented, the structure of the segmentation needs to be recorded in the
+structural metadata.
 
-Furthermore, Figure [1](#fig1) shows that metadata can be stored either at the
-representation level or at the package level. In the CSIP it is explained in more
-detail where different types of metadata can be stored. In this context, it is
-sufficient to mention that descriptive, technical, preservation, and rights
-metadata can relate either to the IP as a whole or to individual representations.
+In [@OAIS2012] p. 1-9, the Archival Information Collection (AIC) is described as  
+“an Archival Information Package whose Content Information is an aggregation of other 
+Archival Information Packages." The AIC can therefore represent a the structure of a segmented 
+AIP is defined by a header information package (AIC) pointing to the child information 
+packages (AIPs). 
 
-### Representations in the AIP
+It is recommended that the structural metadata of the child information packages record
+to which header information package (AIC) they belong. 
 
-As already mentioned, the structural requirements defined by the CSIP generally
-apply to the AIP. However, the AIP `representations` folder contains
-representations which are created as part of the AIP maintenance. If no
-maintenance operation was performed, this folder is not present. For this reason,
-the CSIP requirement which prescribes the presence of the `representations`
-folder is overruled by the following AIP specific requirement.
+If during the life-cycle of the AIP preservation actions are applied to specific parts, 
+i.e. child packages of the logical AIP, the AIC (header information package) must update
+the references to the child packages. However, it is not required to update the 
+reference to a parent of a child package which is not concerned by a preservation action.
 
-<a name="aip-representations-optional"></a>**AIP-REPRESENTATIONS-OPTIONAL**: The
-Archival Information Package (AIP) folder COULD include a folder named
-representations. This requirement overrules requirement *CSIPSTR9* defined by
-the CSIP. See also requirement *AIP-REPRESENTATIONS*.
+### Splitting 
+
+*Definition:* *Splitting* is a special case of segmentation where large files 
+(e.g. large representation content files) are divided into parts of a fixed byte 
+length. However, the splitted content files are wrapped by AIP segments, i.e. they 
+are contained in an AIP which references the parent information package (AIC) 
+to which they belong.
+
```

Comment:
FYI, the definitions of splitting and segmentation are clear to my mind at least :)

Reply:
Good.